packages/connector-local/src/connector.ts

```diff
@@ -300,9 +301,16 @@ export default class LocalConnector implements IConnector {
                 dot: true,
                 gitignore: true
             } as any));
+
+            // Ignore options.content when matching multiple files
+            if (options && options.content) {
+                delete options.content;
```

Comment:
why don't set just to null?

Reply:
No reason other than preference. I'll change this to set `content` to `null` instead.